test/mock/cor-backend/services/question.js

```diff
@@ -27,6 +30,18 @@ module.exports = {
     question_body_text: params => questionData[params.questionId].body,
     answer: params => getCachedAnswer(params.questionId),
     answer_state: params => getCachedState(params.questionId),
-    answer_date: params => getCachedDate(params.questionId)
+    answer_date: params => getCachedDate(params.questionId),
+    evidence: [
+      {
+        id: 'c9c29d3b-9619-4ac5-bfe5-27ad70213e55',
+        file_name: 'doctor.doc',
+        created_date: () => moment.utc().subtract(1, 'day').endOf('day').format()
+      },
+      {
+        id: 'c9c29d3b-9619-4ac5-bfe5-27ad70213e56',
+        file_name: 'doctor.doc',
```

Comment:
Can we name the files differently? I know it doesn't matter but will be more pleasing to the eye 😉

Reply:
did that on purpose, as I consider it a UX issues